internal/pkg/amqp/receiver.go

```diff
@@ -87,6 +98,64 @@ func NewAMQPServer(urlStr string, debug bool, msgcount int, prefetch int) *AMQPS
 	return server
 }
 
+//GetHandler  ...
+func (s *AMQPServer) GetHandler() *AMQPHandler {
+	return s.amqpHandler
+}
+
+//NewAMQPHandler  ...
+func NewAMQPHandler(source string) *AMQPHandler {
+	plabels := prometheus.Labels{}
+	plabels["source"] = source
+	return &AMQPHandler{
+		totalCount:     0,
+		totalProcessed: 0,
+		totalCountDesc: prometheus.NewDesc("sa_collectd_total_amqp_message_recv_count",
```

Comment:
Should these `sa_collectd` items be made dynamically configured? Not really sure.

Reply:
These are pretty static and only a few possible.